[2025-01-08T14:26:03.344-0800] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-08T14:26:03.356-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: CovidE2EPipeDag.create_load_sql_file scheduled__2025-01-01T02:10:00+00:00 [queued]>
[2025-01-08T14:26:03.365-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: CovidE2EPipeDag.create_load_sql_file scheduled__2025-01-01T02:10:00+00:00 [queued]>
[2025-01-08T14:26:03.366-0800] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-08T14:26:03.381-0800] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): create_load_sql_file> on 2025-01-01 02:10:00+00:00
[2025-01-08T14:26:03.384-0800] {standard_task_runner.py:72} INFO - Started process 9299 to run task
[2025-01-08T14:26:03.394-0800] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'CovidE2EPipeDag', 'create_load_sql_file', 'scheduled__2025-01-01T02:10:00+00:00', '--job-id', '1310', '--raw', '--subdir', 'DAGS_FOLDER/CovidE2EPipeDag.py', '--cfg-path', '/var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T/tmpfg4t48jc']
[2025-01-08T14:26:03.396-0800] {standard_task_runner.py:105} INFO - Job 1310: Subtask create_load_sql_file
[2025-01-08T14:26:03.451-0800] {task_command.py:467} INFO - Running <TaskInstance: CovidE2EPipeDag.create_load_sql_file scheduled__2025-01-01T02:10:00+00:00 [running]> on host Matthews-MacBook-Air.local
[2025-01-08T14:26:03.536-0800] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matthew Horn' AIRFLOW_CTX_DAG_ID='CovidE2EPipeDag' AIRFLOW_CTX_TASK_ID='create_load_sql_file' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T02:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T02:10:00+00:00'
[2025-01-08T14:26:03.538-0800] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-08T14:26:03.588-0800] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T
[2025-01-08T14:26:03.599-0800] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'echo "COPY INTO CovidE2EPipeDatabase.raw_data.industrial_production\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/industrial_production_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.stocks_eod\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/stocks_eod_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.currency_exchanges\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/currency_exchanges_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.gdp\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/gdp_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.us_cli\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/us_cli_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.unemployment_rate\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/unemployment_rate_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.thirty_yr_mortgage\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/thirty_yr_mortage_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n\n    COPY INTO CovidE2EPipeDatabase.raw_data.trade_balance\n    FROM @CovidE2EPipeDatabase.raw_data.raw_data_stage/trade_balance_2025-01-08.csv\n    FILE_FORMAT = ( TYPE=\'CSV\' )\n    ON_ERROR = \'CONTINUE\';\n    " > /Users/matthewmac/airflow/helper_scripts/load_staged_data_2025-01-08.sql']
[2025-01-08T14:26:03.608-0800] {subprocess.py:99} INFO - Output:
[2025-01-08T14:26:03.614-0800] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-08T14:26:03.649-0800] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-08T14:26:03.652-0800] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=CovidE2EPipeDag, task_id=create_load_sql_file, run_id=scheduled__2025-01-01T02:10:00+00:00, execution_date=20250101T021000, start_date=20250108T222603, end_date=20250108T222603
[2025-01-08T14:26:03.737-0800] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-08T14:26:03.810-0800] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-08T14:26:03.817-0800] {local_task_job_runner.py:245} INFO - ::endgroup::
